vllm/entrypoints/api_server.py

```diff
@@ -102,6 +103,7 @@ async def stream_results() -> AsyncGenerator[bytes, None]:
         help="FastAPI root_path when app is behind a path based routing proxy")
     parser.add_argument("--log-level", type=str, default="debug")
     parser = AsyncEngineArgs.add_cli_args(parser)
+    parser = TensorizerArgs.add_cli_args(parser)
```

Comment:
Please keep in mind that from `TensorizerArgs.add_cli_args` [was actually initially injected directly in to `AsyncEngineArgs` ](https://github.com/vllm-project/vllm/pull/3476/files#diff-ea8b8ff63961713ccb62d78e53e96404b587b7828cb9fee08a9e5576bf563673R424) in the initial PR adding Tensorizer support to vLLM. #4097 moved `TensorizerArgs` out from this and allowed Tensorizer to be configured from the `model_loader_extra_config` parameter within the `vLLM` object entrypoint, accepting a `TensorizerConfig` object, or `--model-loader-extra-config` via the CLI accepting a JSON string with the desired parameters.